Aptoide/aptoide-client-v8
# [AN-1901] "You might also like" listApps request cdn="pool" to cdn="web"

What does this PR do?

   This pull-request undergoes a small refactor to the CDN parameters used in the v7 requests. The cdn value is now an Enum instead of an hardcoded injection.
   ListApps request in the AppView (when install is clicked, a.k.a "You might also like" request) cdn parameter was changed from "pool" to "web".

Where should the reviewer start?

- [ ] AppViewFragment : loc 927 (requestFactoryCdnWeb)

How should this be manually tested?

  Open Aptoide v8 > AppView > click on install > check charles for the listApps request if cdn = web.

  Check if all requests are sending the Cdn parameter correctly.

What are the relevant tickets?

Tickets related to this pull-request: [AN-1901](https://aptoide.atlassian.net/browse/AN-1901)

Screenshots (if appropriate):
N/A

Questions:

   Is there a blog post? N/A
   Does this add new dependencies which need to be added? No
